.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules/
 .env
 build
 coverage/
+.prettierrc
```

Comment:
@phelanjo Do you have a `.prettierrc` to add to the project?

Reply:
I do! it's super basic though, just sets tab width and single quotes.